conf/drupal/config/field.storage.node.field_migrated_node_id.yml

```diff
@@ -0,0 +1,21 @@
+uuid: ccad061d-64c4-4c4c-ad6a-074c1f25f28b
+langcode: en
+status: true
+dependencies:
+  module:
+    - node
+id: node.field_migrated_node_id
+field_name: field_migrated_node_id
+entity_type: node
+type: string
```

Comment:
Maybe it doesnt matter, but I'm surprised tis isnt an integer field

Reply:
Yeah, agreed. Changed this to an integer field.